user_docs/en/userGuide.t2t

```diff
@@ -2925,6 +2926,30 @@ If you install an add-on with paid components and change your mind about using i
 The Add-on Store is accessed from the Tools submenu of the NVDA menu.
 To access the Add-on Store from anywhere, assign a custom gesture using the [Input Gestures dialog #InputGestures].
 
+++ Note on security and privacy when using Add-ons ++[AddonSecurityandPrivacy]
+Installing add-ons in NVDA leads to integration of external code into NVDA's functionality in order to enhance NVDA or make new features possible.
+Add-ons can also use external libraries and third party services to serve the purpose and provide the features for which they have been developed. They can be developed by every person or company.
+
+Automated code checks for add-ons happen when they are submitted to the NVDA’s official add-on store. This means e.g.  they will undergo  a basic automated check for vulnerabilities by security analysis tools such as codeQL.

```

Comment:
1. Remove "the" with "'s".
2. Usually "Add-on Store" is capitalized
3. I maintain my opinion on removing "official". Here in the user Guide and in any documentation of NV Access, it is clear that "Add-on Store" means NV Access' one. There is no ambiguity. And adding "official" would add more confusion, i.e. some people may ask: "Oh? Didn't know that there were official and unofficial stores? How can I check that the one I am using is official? Where is the unofficial one? etc."


```suggestion
Automated code checks for add-ons happen when they are submitted to NVDA’s Add-on Store. This means e.g.  they will undergo  a basic automated check for vulnerabilities by security analysis tools such as codeQL.
```